README.md

```diff
@@ -1,35 +1,53 @@
-# ExercismWinSetup
+# Windows Installer for Exercism Windows CLI
 A project to download and install the latest version of Exercism Client for Windows depending on the architecture.
 
 Exercism is licensed under MIT License. Copyright (c) 2013 Mike Gehard, Katrina Owen
 
 Exercism Repository -> https://github.com/exercism/exercism.io
 
-[![Build status](https://ci.appveyor.com/api/projects/status/6b5cka3nm3r4vh8d?svg=true)](https://ci.appveyor.com/project/blueelvis/windows-installer)
+[![Build status](https://ci.appveyor.com/api/projects/status/6b5cka3nm3r4vh8d?svg=true)](https://ci.appveyor.com/project/rpottsoh/windows-installer)
 
-#System Requirements  
-  *Operating System* = Windows Vista, Windows 7, Windows 8/8.1 , Windows 10  
-  *Architecture* = 32bit / 64bit both are supported  
+# System Requirements  
+  *Supported Operating Systems* = Windows Vista, Windows 7, Windows 8/8.1 , Windows 10  
+  *Architecture* = 32bit / 64bit  
   *Internet Connection* = Required to fetch the latest Exercism Binary  
   *Hard Disk* = 10MB for Client, 1KB for Configuration and some additional space to store the fetched exercises.  
     
     
     
 
 
-#Step-By-Step Install Procedure =  
- 1. This is the License Agreement form. Accept the agreement to proceed ahead. ![License Agreement](https://github.com/blueelvis/ExercismWinSetup/raw/master/Resources/License%20Agreement.png "License Agreement Form")
- 2. Once you accept the agreement, you are presented with the below screen. Browse or type the location of the folder where you would like to install Exercism. ![Install Location](https://github.com/blueelvis/ExercismWinSetup/raw/master/Resources/Install%20Location.png "Install Location Form")
- 3. After this, there might be a pause. Don't worry everything is working fine and you would soon be presented with the below screen where the Setup downloads the latest version of Exercism CLI from it's Github Repository - ![Client Download Form](https://github.com/blueelvis/ExercismWinSetup/raw/master/Resources/Client%20Download.png "Client Download Form")
- 4. The final step is to enter the Exercism API key which allows you to fetch, keep track of exercises, submit solution etc - ![Configure API Form](https://github.com/blueelvis/ExercismWinSetup/raw/master/Resources/Configure%20API.png "Configure API Form")
- 5. That's it. Click `Finish` and you are done :0)
+# Step-By-Step Install Procedure    
+## License  
+ ![License Agreement](img/installLicense.png)  
+ This is the License Agreement form. To proceed with the installation accept the license agreement by clicking the `Next` button.  
```

Comment:
I still feel quite strongly that we shouldn't have the license agreement as an actual step. In the welcome screen we can say that the license is MIT (and link to it), and that by using the installer, they agree to the license.

Ideally it would start straight on "install location" with just a bit of small text about the license.